### DISCUSSION TITLE
PATCH Problem~~

### DISCUSSION THREAD

**Discussion by vagcom** [Mr-MIBonk/M.I.B._More-Incredible-Bash#152] [Q&A] [answered]
hello
  I did MHI2_KR_AU57x_S0035_MU0035_PATCH but Carplay or AndroidAUTO doesn't appear
What could be the problem?
They say the patch file doesn't work, but is there any way?
please!!

![KakaoTalk_20230319_114655621_07](https://user-images.githubusercontent.com/128269735/226154052-6648ee87-76a7-4864-a982-ffaf6b1ec4f2.jpg)
![KakaoTalk_20230319_114655621_04](https://user-images.githubusercontent.com/128269735/226154054-6ab415e9-2b24-4599-8b85-5191948e3edb.jpg)
![KakaoTalk_20230319_114655621_05](https://user-images.githubusercontent.com/128269735/226154055-4d6ab22c-5661-4ab6-a471-7a672f5ec59b.jpg)
![KakaoTalk_20230319_114655621_06](https://user-images.githubusercontent.com/128269735/226154056-afb9d36c-27cb-4373-85d8-1682c8d53dbb.jpg)
![KakaoTalk_20230319_114655621_01](https://user-images.githubusercontent.com/128269735/226154057-967de587-c343-452d-afcc-94356e0e7c72.jpg)
![KakaoTalk_20230319_114655621_02](https://user-images.githubusercontent.com/128269735/226154059-aedbceee-b07f-4579-93fd-a4aac6997607.jpg)
![KakaoTalk_20230319_114655621_03](https://user-images.githubusercontent.com/128269735/226154060-8a97541a-2bcf-4e1e-bf86-64a7e7a0f550.jpg)
![KakaoTalk_20230319_114655621](https://user-images.githubusercontent.com/128269735/226154062-f4714ee3-f463-40d0-8006-dcfeffa79636.jpg)

**Comment by harman-f** [comment]
That FW is just too old and does not support CP/AA. 
you need to update the FW, however there are no official FW files available.

**Reply by vagcom** [reply]
There were people who didn't have to update the FW...
The patch file doesn't seem to match

**Reply by vagcom** [reply]
![KakaoTalk_20230319_151516785](https://user-images.githubusercontent.com/128269735/226157229-d3284c22-cd3c-410d-bcde-8e53f3daa148.jpg)

**Comment by harman-f** [comment, marked as answer]
I downloaded your backup.
There is one "failed" backup, where M.I.B was not able to extract stuff from FW.
![grafik](https://user-images.githubusercontent.com/98130152/226160341-70b8f339-0b3a-41b6-b3cf-50bddc8d1312.png)
What did you change inbetween?

My understanding is, that this FW does not contain the software parts needed for AA/CP.

There seem to be 2 alternative solutions floating around:
1. Custom FW update to MHI2_KR_AU57x_P0901
2. Partial update of MMX to newer version - at least that is what I assume.

**Reply by vagcom** [reply]
MU0035-allversion.txt seems to be created after patching and I don't know why MU-allversion.txt exists.

Even if I update to MHI2_KR_AU57x_S0035_MU0035_PATCH
There were no changes to the car.

Some people have CarPlay and Android Auto enabled on their cars with the same firmware version.

I think I can patch it with method 2, but it seems to be because the patch file is different.

**Reply by vagcom** [reply]
Maybe FecContainer.fec inside MHI2_KR_AU57x_S0035_MU0035_PATCH is wrong?
